### ISSUE TITLE
Github services deprecation

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#14287]
https://developer.github.com/changes/2018-04-25-github-services-deprecation/

Github services as being deprecated in favor of web hooks. I believe we currently use several services, so we'll have to investigate moving them to web hooks.

**Comment by williamdes** [comment]
The email "service" will be deprecated (AFAIK) https://github.com/phpmyadmin/website/blob/6c8569028c5d1b437ea2bc34190b6fe9eba338de/pmaweb/templates/support.html#L63

**Comment by nijel** [comment]
Yes, email is probably the most problematic, rest can be easily migrated to webhooks or apps (where it exists).

Services we currently have:

- [ ] CodeClimate
- [ ] Email
- [ ] Jenkins
- [ ] ReadTheDocs
- [ ] Travis CI
- [ ] Weblate

**Comment by williamdes** [comment]
@nijel Self hosted email sender ?

Who has the list of subscribed emails ?

What are the uses of email service, only notifications ?
Using rss https://github.com/phpmyadmin/phpmyadmin/commits/master.atom ?

I thought about this issue when I found https://git-scm.com/docs/git-send-email

**Comment by nijel** [comment]
It goes to git mailing list: https://lists.phpmyadmin.net/mailman/listinfo/git

**Comment by williamdes** [comment]
So a web-hook to server that creates and sends an email to mailing list should be OK ?

I remember subscribing to mailing list but I receive mails with a GitHub footer, I am not sure to understand correctly the actual system.

Is GitHub sending an email to mailing list ? like : https://help.github.com/enterprise/2.13/admin/guides/user-management/configuring-email-for-notifications/
 
>**NOTE:** This service been marked for deprecation: https://developer.github.com/changes/2018-04-25-github-services-deprecation/ 

> Functionality will be removed from GitHub.com on January 31st, 2019.

**Comment by nijel** [comment]
Yes, GitHub is currently generating the emails and sends them to the mailing list where anybody can subscribe.

**Comment by williamdes** [comment]
Ok, do you think I can work on finding a way to resolve this issue ?
Hosted web server who waits for web-hooks and does the generating & sending emails ?
NodeJS ?
PHP ?
I will use twig in both cases to generate emails from a template.

**Comment by nijel** [comment]
We already do have server side hooks, so it's probably good idea to integrate it there: https://github.com/phpmyadmin/scripts/tree/master/hooks

**Comment by williamdes** [comment]
@nijel /close ?

**Comment by nijel** [comment]
I've just replaced email service with hook @williamdes implemented.

**Comment by mauriciofauth** [comment]
@nijel Packagist is indicating that phpMyAdmin packages are using legacy auto-update. According to [this tweet](https://twitter.com/packagist/status/1037035587453235201), it's necessary to logout/login with GitHub to upgrade the hooks.

![pma](https://user-images.githubusercontent.com/120970/47811782-64d4db80-dd25-11e8-9534-bc5d075351a5.png)

**Comment by nijel** [comment]
I already did that, but that doesn't seem to change anything:

![screenshot_2018-10-31 packagist](https://user-images.githubusercontent.com/212189/47817925-3839b780-dd57-11e8-8f9e-59638c1e0cc8.png)

**Comment by williamdes** [comment]
@nijel I had to disconnect my account from github and connect back (see screenshot).
Is that what you did ?
![Packagist](https://user-images.githubusercontent.com/7784660/47818703-c5c9d700-dd58-11e8-85de-f6e6a4998d9a.jpg)

**Comment by nijel** [comment]
There was no account connected to phpmyadmin account, so I did just connect.

**Comment by williamdes** [comment]
@nijel And manual hook sync ?

**Comment by nijel** [comment]
The hook *is* there, it's just packagist still showing legacy label. Probably safe to ignore and will hopefully disappear at some point...

**Comment by williamdes** [comment]
@nijel Okay, had the same issue with mine and after random operations the warning disappeared ...

**Comment by mauriciofauth** [comment]
Now, _phpmyadmin/phpmyadmin_ is ok, but the other packages are still showing the legacy label. Maybe it take some time to update.

![phpmyadmin](https://user-images.githubusercontent.com/120970/47827437-b3519c80-dd5c-11e8-9c8a-30e0e14d40cd.png)
